src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -103,7 +103,7 @@
   <Target Name="IncludeProjectToProjectAssets">
     <ItemGroup>
       <BuildOutputInPackage Include="$(ProjectDir)bin\$(Configuration)\net45\Design\Microsoft.Xaml.Behaviors.DesignTools.dll"
-                            Condition="'$(TargetFramework.StartsWith('netcoreapp3.'))">
+                            Condition="$(TargetFramework.StartsWith('netcoreapp3.')) or $(TargetFramework.StartsWith('net5'))">
```

Comment:
May I suggest the following as being more reliable than a startswith check. I'll also work when net6.0-windows is added without having to be changed.
```suggestion
                            Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '3.0'))">
```

Reply:
Good suggestion. That's how we do it internally for the designer in VS as well.

Reply:
@mgoertz-msft I've changed it.